modules/deployer-bootstrap.nix

```diff
@@ -0,0 +1,63 @@
+{ config, lib, pkgs, ... }:
+
+# This is a minimal NixOS configuration for the deployer so that
+# devops can login and run iohk-ops to fully deploy the deployer and
+# then other hosts.
+
+let
+  username = "deployer";
+
+in {
+    imports = [
+      <nixpkgs/nixos/modules/virtualisation/amazon-image.nix>
+    ];
+    ec2.hvm = true;
+
+    users.users.${username} = {
```

Comment:
I think we should reuse keys from lib.nix

Reply:
Yeah I also pondered that when I saw this. The problem is that this is passed to ec2 instances as `user-data` which is a metadata field (with a limited size) that can be used to pass arbitrary data to the AMI. The NixOS AMI uses this as its `configuration.nix`. Point is that this file ends there alone so it can't just `import ../lib/ssh-keys.nix` so this requires some thought. I had a few ideas, some dumber than others, but most involved more engineering acrobatics than I felt this was worth. tl;dr: I think the simplest solution here would be to simply remove all the keys in favour of a single one: bootstrapping is a one off operation. If I understand correctly how this works, then this config is temporary and its sole purpose is to prepare the instance for NixOps to scp a closure and activate it. Should be pretty straight forward to modify `gac-mkdeployer.sh` to template-in a public key (either provided by the user or generated. probably doesn't matter); question is whether this should be scoped here or attacked separately later.

Reply:
This is arguably the simplest solution -- the downside cost being people accidentally able/unable to deploy a new deployer (which is independently controlled by AWS).  So the real cost, in the end, is inability of new members to deploy a new deployer, which isn't a terribly frequent situation.

Any other costs I'm missing?

If not, is it worth complicating things in a non-trivial way just to streamline this single case?